### DISCUSSION TITLE
Center mouse pointer

### DISCUSSION THREAD

**Discussion by vmv** [tomasklaen/uosc#469] [Ideas]
I really need a simple option like this.. to center the mouse pointer after a certain amount of inactivity.. like 5 seconds (menu also dissapear when pointer is centered).
Why,.. because i pause/start the play with mouse..:).. and it's always moving slowly to menu activation area.

Nothing important ofc...

thanks!

**Comment by po5** [comment]
Moving the mouse isn't something a simple lua script can do.  
Sounds trivial to implement as its own mpv script that calls xdotool/Windows equivalent. Out of scope.
